pkg/controller/jobs/appwrapper/appwrapper_controller.go

```diff
@@ -128,11 +130,28 @@ func (aw *AppWrapper) PodSets() ([]kueue.PodSet, error) {
 	}
 	podSets := make([]kueue.PodSet, len(podSpecTemplates))
 	for psIndex := range podSpecTemplates {
+		var podIndexLabel *string
```

Comment:
Could you expand UTs? https://github.com/kubernetes-sigs/kueue/blob/6d1ccd92f0ff088303e9cbf18d1d1565e2f40795/pkg/controller/jobs/appwrapper/appwrapper_controller_test.go#L50

I think that PodSets UTs for other jobs are helpful in understanding what we should add.

Reply:
Done.